lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -668,6 +680,8 @@ export class UserAgentApplication {
                 request.correlationId,
             );
 
+            this.logger.verbose("Finished building server authentication request");
+
             // populate QueryParameters (sid/login_hint) and any other extraQueryParameters set by the developer
             if (ServerRequestParameters.isSSOParam(request) || account) {
                 serverAuthenticationRequest.populateQueryParams(account, request, null, true);
```

Comment:
Can we add a log statement inside this `if`?

Reply:
Added log statement. Let me know if you want this statement changed.